tests/Unit/Domain/CoordinateMaps/KerrHorizon.py

```diff
@@ -0,0 +1,15 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+
+def call(coords, spin):
+    for coord in coords:
+        print(coord)
```

Comment:
Are these `print`s left over from debugging?

Reply:
I was sure I had deleted that file, not sure how it snuck back in